api/handlers/websocket-handler.go

```
@@ -0,0 +1,631 @@
+	}
+}
+
+// readPump maneja los mensajes entrantes del cliente
+func (c *Client) readPump() {
+	defer func() {
+		c.Hub.Unregister <- c
+		c.Conn.Close()
+	}()
+
+	c.Conn.SetReadLimit(512)
+	c.Conn.SetReadDeadline(time.Now().Add(60 * time.Second))
```

golangci-lint (1.64.8)

Error return value of `c.Conn.SetReadDeadline` is not checked

(errcheck)